### PR TITLE
`FusionExecutor::runRtc` don't use `getBuffer`

### PR DESCRIPTION
**Pull request by zasdfgbnm**
Because it is deprecated in https://github.com/NVIDIA/Fuser/pull/644

### CODE REVIEW

**Comment by zasdfgbnm** [comment]
!build

File: csrc/executor_kernel_arg.cpp [hunk 1]
```diff
@@ -647,4 +647,46 @@ void KernelArgumentHolder::pushTensorProxy(
   arguments_.push_back(getAbstractTensorArg(at::Tensor(meta_tensor)));
 }
 
+std::vector<std::byte> getTensorArgBuffer(
```
Comment (zasdfgbnm) [inline csrc/executor_kernel_arg.cpp:650]: This will be used to replace `getTensorArg` in the future.

**Comment by zasdfgbnm** [comment]
!build

**Review by naoyam — APPROVED** [review]
LGTM

**Pull request merged**